app-text/sgml-common/sgml-common-0.6.3-r6.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils prefix
+
+DESCRIPTION="Base ISO character entities and utilities for SGML"
+HOMEPAGE="http://www.iso.org/standard/16387.html"
+#SRC_URI="mirror://kde/devel/docbook/SOURCES/${P}.tgz"
+SRC_URI="https://dev.gentoo.org/~floppym/dist/${PN}/${P}-gentoo.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
+
+src_prepare() {
+	# We use a hacked version of install-catalog that supports the ROOT
+	# variable, puts quotes around the CATALOG files, and can be prefixed.
+	cp "${FILESDIR}/${P}-install-catalog.in" "${S}/bin/install-catalog.in"
+
+	epatch "${FILESDIR}"/${P}-prefix.patch
+	eprefixify bin/install-catalog.in bin/sgmlwhich config/sgml.conf
+	default
+}
+
+src_configure() {
+	econf --htmldir="${EPREFIX}/usr/share/doc/${PF}/html"
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "emake install failed"
+}
+
+pkg_postinst() {
+	local installer="${EROOT}usr/bin/install-catalog"
+	if [[ ! -x ${installer} ]]; then
+		eerror "install-catalog not found! Something went wrong!"
+		die "install-catalog not found! Something went wrong!"
+	fi
+
+	einfo "Installing Catalogs..."
+	"$installer" --add \
+		"${EPREFIX}"/etc/sgml/sgml-ent.cat \
```

Comment:
first you're using `EROOT`, but `EPREFIX` here? not consistent, use `${EROOT%/}` here too

Reply:
Not sure whether I understand this completely. If someone emerges this package in a prefixed environment I suppose EROOT and EPREFIX will be different. Will it still behave correctly then, if I use EROOT both times?